google/cloud/forseti/notifier/notifier.py

```diff
@@ -145,6 +147,11 @@ def run(inventory_index_id,
                 if not resource['should_notify']:
                     LOGGER.debug('Not notifying for: %s', resource['resource'])
                     continue
+                if notifier_configs.get('email_connector_config'):
```

Comment:
Does this mean that every user will get the email by default?  There's no way to disable emails?

Reply:
If 'should_notify' is set to true and if connector details are not provided in 'email_connector_config' and in 'email_violations' then email will not be sent out.

Reply:
If someone wants to:
- get inventory summary by email, so `email_connector_config` is populated
- get the violations by slack, so `should notify` is populated

Will they still get the email of the violations?

Reply:
- If the `email_connector_config` is populated, they will receive an email of the violations for scanners that are turned on for notification. 
- The same config will be used to send out inventory summary as well without them having to duplicate connector in `email_summary`.
- However if they do not wish to receive inventory summary then they need to disable it as shown below:
```
    email_summary:
         enabled: false
```
- Conversely, if they want to receive only inventory summary by email and not violations then they need not fill out the `email_connector_config` section. Instead they need the `email_summary`.

The aim of this design is to abstract out connector info from all scanners at a common place. I have added the part to send out inventory summary using that info. However if they want only inventory summary and no violations by email they need the 'email_summary' section at least for now.

Thanks for bringing this up!